region/src/main/java/org/motechproject/nms/region/domain/HealthBlock.java

```diff
@@ -38,11 +40,14 @@
     private String hq;
 
     @Field
-    @Unique
     @Column(allowsNull = "false")
     @NotNull
     private Long code;
 
+    // used by CSV import as a placeholder for the taluka
+    @Ignore
+    private String talukaCode;
```

Comment:
I would avoid adding this kind of fields to entity classes (looks like a code smell). If such a placeholder is really needed, consider creating an intermediate class or use a map to store entity properties.